src/backend/tests/crawler/test_crawler.py

```diff
@@ -24,22 +24,22 @@ def test_search_function():
     assert len(record.cases[1].charges) == 1
     assert len(record.cases[2].charges) == 3
 
```

Comment:
Since it's not clear in-line why the expected re-ordering happened, I would add this comment:

```
# cases have been ordered reverse-chronologically and are now in this order:
# X0003(3/6/2018), X0001(3/14/2017) X0002(4/12/1992)
```

But in fact this test case isn't resulting in the output order, the current order here (and the one being tested for, making the test pass) is: `X0003,  X0002, X0001`